src/chess/uciloop.cc

```diff
@@ -215,6 +228,13 @@ void UciLoop::SetLogFilename(const std::string& filename) {
   }
 }
 
+void UciLoop::WriteDebugLogLine(const std::string& line) {
+  static std::mutex logging_mutex;
+  std::lock_guard<std::mutex> lock(logging_mutex);
+  if (debug_log_)
+    debug_log_ << GetTimestamp() << ' ' << line << std::endl << std::flush;
```

Comment:
`std::endl` does `std::flush` itself (unlike outputting '\n'), so no need to flush explicitly.